selfdrive/car/toyota/values.py

```diff
@@ -136,7 +136,7 @@ def check_ecu_msgs(fingerprint, ecu):
   },
   # 2018 Highlander LE
   {
-    36: 8, 37: 8, 114: 5, 119: 6, 120: 4, 170: 8, 180: 8, 186: 4, 426: 6, 452: 8, 464: 8, 466: 8, 467: 8, 544: 4, 545: 5, 550: 8, 552: 4, 608: 8, 610: 5, 643: 7, 705: 8, 725: 2, 800: 8, 835: 8, 836: 8, 849: 4, 869: 7, 870: 7, 896: 8, 916: 3, 921: 8, 944: 8, 945: 8, 951: 8, 955: 8, 956: 8, 979: 2, 1008: 2, 1017: 8, 1020: 8, 1041: 8, 1056: 8, 1059: 1, 1114: 8, 1176: 8, 1177: 8, 1178: 8, 1179: 8, 1180: 8, 1181: 8, 1182: 8, 1183: 8, 1189: 8, 1190: 8, 1191: 8, 1192: 8, 1196: 8, 1197: 8, 1198: 8, 1199: 8, 1206: 8, 1212: 8, 1227: 8, 1279: 8, 1408: 8, 1409: 8, 1410: 8, 1552: 8, 1553: 8, 1554: 8, 1556: 8, 1557: 8, 1561: 8, 1562: 8, 1568: 8, 1569: 8, 1570: 8, 1571: 8, 1572: 8, 1592: 8, 1593: 8, 1595: 8, 1599: 8, 1745: 8, 1779: 8, 1872: 8, 1880: 8, 1984: 8, 1988: 8, 1992: 8, 1996: 8
+    36: 8, 37: 8, 114: 5, 119: 6, 120: 4, 170: 8, 180: 8, 186: 4, 238: 4, 426: 6, 452: 8, 464: 8, 466: 8, 467: 8, 544: 4, 545: 5, 550: 8, 552: 4, 608: 8, 610: 5, 705: 8, 725: 2, 740: 5, 800: 8, 849: 4, 871: 2, 896: 8, 916: 3, 921: 8, 944: 8, 945: 8, 951: 8, 955: 8, 956: 8, 979: 2, 1008: 2, 1017: 8, 1020: 8, 1042: 8, 1044: 8, 1056: 8, 1059: 1, 1114: 8, 1161: 8, 1162: 8, 1163: 8, 1176: 8, 1177: 8, 1178: 8, 1179: 8, 1180: 8, 1181: 8, 1182: 8, 1183: 8, 1189: 8, 1190: 8, 1191: 8, 1192: 8, 1196: 8, 1197: 8, 1198: 8, 1199: 8, 1206: 8, 1212: 8, 1235: 8, 1279: 8, 1408: 8, 1409: 8, 1410: 8, 1552: 8, 1553: 8, 1554: 8, 1556: 8, 1557: 8, 1561: 8, 1562: 8, 1568: 8, 1569: 8, 1570: 8, 1571: 8, 1572: 8, 1592: 8, 1593: 8, 1595: 8, 1599: 8, 1745: 8, 1779: 8, 1872: 8, 1880: 8, 1984: 8, 1988: 8, 1992: 8, 1996: 8
```

Comment:
The command message is bigger from previous commit but I have noticed Lane Departure Alert Malfunction and Headlight System Malfunction alert come very often. The previous command work out of box. I would cherry pick the previous commit but here both messages I got.